### DISCUSSION TITLE
Intersection of Project tags in Run-many with projects and tags instead of `or`

### DISCUSSION THREAD

**Discussion by zackarydev** [nrwl/nx#23276] [Feature Requests]
- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md))

## Description
I want to be able to run projects based on tags. Currently you can run this, but it uses an `or` operator.

Example: `nx run-many --target=serve --projects:tag:application:x,tag:type:api`

This would run all projects tagged `application:x` AND it would run all projects with `type:api`, but it would also run all projects with `type:frontend`

## Motivation
More flexibility in how apps are ran.

## Suggested Implementation
example: `nx run-many --target=serve --projects:tag:application:x&tag:type:api`

**Comment by FrozenPandaz** [comment]
I've converted this to a discussion so it can be discussed further.
